Certification/Loader.php

```diff
@@ -86,13 +106,13 @@ static public function count()
      *
      * @return array
      */
-    static protected function prepareFromYaml(array $categories = array(), $path)
+    protected static function prepareFromYaml(array $categories = array(), $path)
```

Comment:
I'm so busy theses times, sorry ... but my curiosity forces me to ask you: why reverse ``static`` and ``public`` ? :D